tests/distro-regression/tests/tests.yaml

```diff
@@ -88,13 +90,20 @@ tests_options:
       smoke: True
     keystone_v3_smoke_focal:
       smoke: True
+      include-list:
+        - "manila_tempest_tests.tests.api.admin.test_admin_actions.AdminActionsTest.*"
+        - "manila_tempest_tests.tests.api.admin.test_share_instances.ShareInstancesTest.*"
+        - "manila_tempest_tests.tests.api.admin.test_share_snapshot_instances.ShareSnapshotInstancesTest.*"
+        - "manila_tempest_tests.tests.api.admin.test_share_types.ShareTypesAdminTest.*"
+        - "manila_tempest_tests.tests.api.admin.test_shares_actions.SharesActionsAdminTest.*"
   force_deploy:
     - trusty-icehouse
     - trusty-mitaka
     - groovy-victoria
     - hirsute-wallaby
     - impish-xena
     - jammy-yoga
+    - kinetic-zed
```

Comment:
should jammy-zed be added to the list?

Reply:
jammy at this point should be correctly managed, if a charm is not advertising support for it then it's a bug.